srcpkgs/swaylock-effects/patches/meson.build.patch

```diff
@@ -0,0 +1,10 @@
+﻿--- meson.build-orig	2021-01-17 22:22:48.531582216 +0100
++++ meson.build		2021-01-17 22:23:19.891581408 +0100
+@@ -18,7 +18,6 @@
+ 		'-Wundef',
+ 		'-Wvla',
+ 		'-fopenmp',
+-		'-mtune=native',
```

Comment:
Having this in cflags doesn't fill me with confidence about upstream...

Reply:
Thanks for linking me to that article. I haven't really thought about it from a security perspective. It's quite eye-opening and raises valid points.